src/pages/index.js

```diff
@@ -0,0 +1,264 @@
+import "./index.css";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+import { handleSubmit } from "../utils/helpers";
+import Api from "../utils/Api.js";
+
+let selectedCard, selectedCardId;
+
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a95e29ec-5559-4960-bb91-fb0cb43f5997",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, users]) => {
+    cards.reverse();
+    cards.forEach((card) => {
+      renderCard(card);
+    });
+
+    profileImage.src = users.avatar;
+    profileName.textContent = users.name;
+    profileDescription.textContent = users.about;
+  })
+  .catch(console.error);
+
+// Universal elements
+const closeButtons = document.querySelectorAll("#modal-close-btn");
+
+// Profile elements
+const editProfileButton = document.querySelector(".profile__edit-btn");
+const profileImage = document.querySelector(".profile__avatar");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const cardModalButton = document.querySelector(".profile__add-btn");
+const avatarModalButton = document.querySelector(".profile__avatar-btn");
+
```

Comment:
* The `overlay` of the **image modal** is not spread on the whole screen on 320px https://snipboard.io/T7pN30.jpg 
* Please, make the `image modal` match the design on 320px (mobile). See how it looks in figma https://snipboard.io/VGcDLv.jpg . It should be as **wide** as a `card`
* Please, make the `confirmation` modal match the `mobile` design on 320px
* Please, make all modals match the `mobile` design. They should have some paddings (margins) from the sides of the screen https://snipboard.io/bgPGJ0.jpg